web/templates/_includes/search_form.html

```diff
@@ -6,15 +6,13 @@ <h1 class="display-4">
         </h1>
     </div>
     <div class="col-5 d-flex align-items-center justify-content-end">
-        {% if request.user|has_group:"daisy-data-steward" %}
+        {% if request.user|has_group:"daisy-data-steward" and not request.user.is_superuser%}
```

Comment:
I think this condition means Vilem cannot see the button. Superusers are not datastewards by default, and this condition is specifically removing superuser from the list of people allowed to see the import and export buttons